include/zephyr/net/conn_mgr_connectivity.h

```diff
@@ -272,6 +278,8 @@ struct conn_mgr_conn_binding {
  *
  * Non-Blocking.
  *
+ * TODO: Talk about persistence and timeouts in this PR for docs.
```

Comment:
I don't think I understand this TODO.

Reply:
I meant to remove that TODO, I'm not quite sure how it wound up back in the source code, I ran a search for TODO messages at one point. Good catch